apps/core/scenes/results/Results.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
+import { StyleSheet, Text } from '@kiwicom/universal-components';
+
+import type { ResultsList as ResultsListType } from './__generated__/ResultsList.graphql';
+import ResultsList from './ResultsList';
+
+type Props = {|
+  +travelFrom: string,
+  +travelTo: string,
+  +dateFrom: string,
+  +dateTo: string,
+|};
+
+type ResultsType = {|
+  +searchItineraries: ResultsListType,
+|};
+
+export default class Results extends React.Component<Props> {
+  renderInner = (props: ResultsType) => {
+    const { searchItineraries } = props;
```

Comment:
You can handle it in the shortened version:
```
 renderInner = ( { searchItineraries }: ResultsType) => {
....
```
but it is just an idea., nothing big deal.